sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumerOptions.cs

```diff
@@ -112,15 +112,15 @@ public int PrefetchCount
         }
 
         /// <summary>
-        ///   Indicates whether or not the consumer requests metrics for the state of its
+        ///   Indicates whether or not the consumer is expected to keep receive information on the last enqueued event on its
```

Comment:
This reads a bit awkward to me.   Could we consider revising to something like:

```csharp
/// <summary>
///     Indicates whether or not the consumer should request information on the last enqueued event on its
///     associated partition, and track that information as events are received.
/// </summary>
///
/// <value><c>true</c> if information about the partition's last event should be requested and tracked; otherwise, <c>false</c>.</value>
///
/// <remarks>
///   When information about the partition's last enqueued event is being tracked, each event received from the Event Hubs
///   service will carry metadata about the partition that it otherwise would not. This results in a small amount of
///   additional network bandwidth consumption that is generally a favorable trade-off when considered
///   against periodically making requests for partition properties using the Event Hub client.
/// </remarks>
///
```
I'd have been more fine grained in highlighting changes, but I made some small tweaks to verbiage in each of the areas.